src/main/java/frc/robot/commands/drive/DriveFieldOrientedHeadingSnapping.java

```diff
@@ -27,11 +28,27 @@
 public class DriveFieldOrientedHeadingSnapping extends Command {
   private Drive drive;
   private XboxController controller;
+  private final Supplier<Double> xScaledSupplier, yScaledSupplier, zScaledSupplier;
```

Comment:
Rename these to xSupplier, ySupplier, etc. In this context we don't care that the controller axes are scaled.